modules/dnn/perf/perf_net.cpp

```diff
@@ -324,10 +346,9 @@ PERF_TEST_P_(DNNTestNetwork, ViTTrack) {
     randu(inp2, 0.0f, 1.0f);
     inp1 = blobFromImage(inp1, 1.0, Size(), Scalar(), false);
     inp2 = blobFromImage(inp2, 1.0, Size(), Scalar(), false);
-    processNet("", "dnn/onnx/models/vitTracker.onnx", {std::make_tuple(inp1, "template"), std::make_tuple(inp2, "search")});
+    processNet("", "dnn/onnx/models/object_tracking_vittrack_2023sep.onnx", {std::make_tuple(inp1, "template"), std::make_tuple(inp2, "search")});
```

Comment:
Need to change test name too: https://github.com/opencv/opencv/blame/4.9.0/modules/dnn/perf/perf_net.cpp#L419-L419C41